pkg/controllers/vdb/offlineupgrade_reconciler.go

```diff
@@ -151,6 +153,15 @@ func (o *OfflineUpgradeReconciler) Reconcile(ctx context.Context, _ *ctrl.Reques
 	return ctrl.Result{}, o.Manager.logUpgradeSucceeded(sandbox)
 }
 
+func (o *OfflineUpgradeReconciler) saveRestorePoint(ctx context.Context) (ctrl.Result, error) {
+	actor := MakeSaveRestorePointReconciler(o.Rec, o.Vdb, o.Log, o.PFacts, o.Dispatcher, o.Rec.GetClient())
```

Comment:
@roypaulin is this the proper way to call the other Reconciler and its method? 
MakeSaveRestorePointReconciler takes a VerticaDBReconciler which does not exist here, should I also create a VerticaDBReconciler here and pass it down to this make method?